src/components/code-editor/code-editor.tsx

```
@@ -333,8 +345,32 @@ export class CodeEditor {
+        );
+
         return (
             <Host>
+                {hasContent && (
+                    <limel-icon-button
+                        label={copyLabel}
+                        icon="copy"
+                        onClick={this.copyCodeToClipboard}
+                        elevated={true}
+                        aria-label="Copy to clipboard"
+                    />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)